src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+
```

Comment:
Should this extra empty line here be removed?

Reply:
amended as of 4030096e42b9fe4bd5c9ccecfed49e72b055c620

Reply:
amended as of 734dcb9b1359b25907ac79f9f742e0c796f93a03